be/src/exprs/time_functions.cpp

```diff
@@ -1566,6 +1568,55 @@ StatusOr<ColumnPtr> TimeFunctions::from_unix_to_datetime_ms_64(FunctionContext*
     return _t_from_unix_to_datetime_ms<TYPE_BIGINT>(context, columns);
 }
 
+static inline int64_t impl_hour_from_unixtime(int64_t unixtime) {
+    // return (unixtime % 86400) / 3600;
+    static const libdivide::divider<int64_t> fast_div_3600(3600);
+    static const libdivide::divider<int64_t> fast_div_86400(86400);
+    int64_t hour = (unixtime - unixtime / fast_div_86400 * 86400) / fast_div_3600;
```

Comment:
what's performance between regular div and this lib?

Reply:
according to this [doc](https://github.com/ridiculousfish/libdivide), it's 5~10x faster compared with regular DIV instruction.